main/org.thingpedia.iot.climate/dataset.tt

```diff
@@ -1,60 +1,60 @@
-dataset @org.thingpedia.iot.thermostat language "en" {
-    action (p_value :Measure(C)) := @org.thingpedia.iot.thermostat.set_target_temperature(value=p_value)
+dataset @org.thingpedia.iot.climate language "en" {
```

Comment:
If we rename the class, we also need to update the datasets (paraphrase.tsv, annotated.txt) everywhere.

Reply:
let's see, I can do that, if we want to be compliant with home assistante integration name, otherwise we keep thermostat no problem, an I will reverse manifest and dataset of the device

Reply:
Note that, iirc, Home Assistant Climate is Temperature + Humidity + Thermostat, and we also have io.home-assistant.climate that does that. You'll need to bring that into main/ too I think.